terms/hyperparameters_nm.tex

```diff
@@ -0,0 +1,70 @@
+\chapter[Hyperparameters]{Hyperparameters}
+\label{ch:Hyperparameters}
+
+As stated by Li \textit{et al.} in \cite{Li2017}, "performance of machine learning algorithms depends critically on identifying a good set of hyperparameters". Hyperparameter tuning is a topic often discussed in literature describing any combination of machine learning algorithms. Adequate values set for hyperparameters will massively influence training of a model, be it in terms of processing time, as well as the actual robustness of the model in question. 
```

Comment:
Tighter writing "\citet{Li2017} state that"
For double quotes in latex use `` '' (not double quote character)
Hyperparam values dont only influence training but also testing!